src/web/App.mjs

```diff
@@ -160,7 +160,12 @@ class App {
         // has completed.
         if (this.autoBakePause) return false;
 
-        if (this.autoBake_ && !this.baking) {
+        if (this.baking) {
```

Comment:
I do intend to review this properly in time, I just think this PR likely needs an extensive amount of testing to make sure it doesn't break any other aspects of the service.

In particular, I'm struggling to work out why the original implementation _explicitly excluded_ autobaking when an existing value was being computed.

As well as that, I'm worried there's a race case somewhere here.  Cancelling a bake in a worker looks asynchronous, as far as I can tell.

I should have more time tomorrow night to look into this, although I generally have more time for this project over the weekends.